sample-app/src/main/java/com/splunk/android/sample/FirstFragment.java

```diff
@@ -184,12 +188,22 @@ public LiveData<String> getSessionId() {
     public void onDestroyView() {
         super.onDestroyView();
         binding = null;
+
+        SplunkRum.getInstance().setScreenName(null);
```

Comment:
I think we want to keep the existing Fragments unchanged and if anything make a new one to demonstrate the behavior.

Reply:
I just added a link to my new activity, where I added Jetpack Compose Navigation, so I think this is now addressed.